docs/source/release_notes/release-notes.rst

```diff
@@ -43,6 +43,15 @@ develop
          - Change
 
     *    - |fixed|
+         - Fixed schema generation with Java 8 optionals.
+           To use Java8 optionals, supply ``OptionalType.JAVA8`` as an additional constructor argument when creating your ``Schema`` object.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1501>`__)
+
+    *    - |devbreak|
+         - The above fix requires all projects to regenerate their schemas, even if not using Java 8 optionals.
```

Comment:
Should we mention a part of the mechanics of the fix (in particular, that the `AtlasDbNamedImmutableTable` and `AtlasDbNamedMutableTable` interfaces have been removed)? From ticket #1249 no internal product uses this but others might.